packages/lambda-tiler/src/import/imagery.find.ts

```diff
@@ -0,0 +1,54 @@
+import { AwsCredentials } from '@chunkd/source-aws-v2';
+import { fsa } from '@chunkd/fs';
+import { Env } from '@basemaps/shared';
+
+export type RoleConfig = {
+  bucket: string;
+  accountId: string;
+  roleArn: string;
+  externalId?: string;
```

Comment:
```typescript
/** Duration in seconds for how long the role can be assumed */
roleSessionDuration?: number
```

Reply:
default 3600?

Reply:
Doesn't need to handle default value here as `AwsCredentials.fsFromRole` already handle it to 3600 default for us.